### PR TITLE
added new system.data.common* apis

### PR DESCRIPTION
**Pull request by mairaw**
/cc @JRAlexander @JennieHubbard

@saurabh500, @corivera can one of you review this?

Internal review URLs:
[DbColum](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.common.dbcolumn?view=netcore-2.0&branch=pr-en-us-2953)
[DbDataReaderExtensions](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.common.dbdatareaderextensions?view=netcore-2.0&branch=pr-en-us-2953)
[I​Db​Column​Schema​Generator](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.common.idbcolumnschemagenerator?view=netcore-2.0&branch=pr-en-us-2953)

### CODE REVIEW

**Comment by mairaw** [comment]
@karelz can you make sure I've requested review from the right folks here?

**Comment by karelz** [comment]
@corivera @divega can you please code review? cc @stephentoub

File: xml/System.Data.Common/DbColumn.xml [hunk 1]
```diff
@@ -109,8 +109,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the base column name; otherwise,                  <see langword="null" /> if no value is set. Can be set to either the column name             or <see langword="null" /> when overridden in a derived class.</summary>
```
Comment (divega) [inline xml/System.Data.Common/DbColumn.xml]: Is the white space expected?

File: xml/System.Data.Common/DbColumn.xml [hunk 2]
```diff
@@ -550,8 +550,8 @@
         <ReturnType>System.Nullable&lt;System.Int32&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the numeric precision of the column data; otherwise,                  <see langword="null" /> if no value is set. Can be set to either an                 <see langword="int32" /> value to specify the numeric precision of the column data or                   <see langword="null" /> when overridden in a derived class.</summary>
```
Comment (divega) [inline xml/System.Data.Common/DbColumn.xml]: White space

File: xml/System.Data.Common/DbColumn.xml [hunk 3]
```diff
@@ -550,8 +550,8 @@
...
+        <value>An                 <see langword="int32" /> value that specifies the precision of the column data, if the data is numeric; otherwise, a null reference (                  <see langword="Nothing" /> in Visual Basic) if no value is set.</value>
```
Comment (divega) [inline xml/System.Data.Common/DbColumn.xml]: White space

**Review by divega — APPROVED** [review]
LGTM besides the extra white space.

**Comment by corivera** [comment]
Is this just a documentation-only change?

**Comment by mairaw** [comment]
Yes @corivera, this is to add docs to https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbcolumn?view=netcore-2.0 and other apis from this PR which are currently empty.

@divega good catch on the spaces. This is a problem exporting the content from our old tool to GitHub. I'll fix that.

**Review by corivera — APPROVED** [review]

**Pull request merged**